01/implement-most-frequent-k-elements.instructor.md

```diff
@@ -0,0 +1,29 @@
+# Instructor: Implement Most Frequent K Elements
+
+An example of a working implementation:
+
+```python
+def top_k_frequent_elements(arr, k):
+    if len(arr) == 1: return [arr[0]]
+    
+    hash_table = {}
+    result = []
+
+    for num in arr:
+        if hash_table.get(num, False):
```

Comment:
I thought about using 0 but I chose to go with False just in case 0 was an integer included in the given array. Also, I didn't know that `if key in dict` had a time complexity of O(1)! Thank you!